keylime.conf

```diff
@@ -193,13 +193,16 @@ registrar_private_key_pw = default
 # See this document https://keylime-docs.readthedocs.io/en/latest/installation.html#database-support
 # for instructions on using different database configurations.
 # The default is "sqlite" and will be located at "/var/lib/keylime/cv_data.sqlite".
+# database_url takes precedence and when set, other database fields are ignored.
```

Comment:
Can we have an example sqlite value for database_url in the comments? Or as the default?

Reply:
NP to document in the comments, but make it default needs to remove the special logic on root/non-root because they use different paths. Sinc we agree to make verifier/registrar running as non-root, I guess we can just go ahead and set the fixed path to /var/lib/keylime/cv_data.sqlite?

Reply:
Yeah, we want to remove running as root for verifier/registrar but I don't want to hold this change up for that. So maybe just documenting it in the comments will be good enough for now.